### PR TITLE
fix "remove it." to "remove it?"

### PR DESCRIPTION
**Pull request by justinwoo**
This has been bothering me for a while, so I made a PR for it.

### CODE REVIEW

**Comment by ndmitchell** [comment]
Thanks for highlighting the issue, I can see why it rankles you - we should definitely fix it.

However, I wonder if that is the right fix? What I'm trying to express is:

```
Found:
  old code
Why not:
  new code
```

But where there is no `new code`. Is the `Why not` in the normal case, without ?, annoying? It seems weird to have question marks for one but not the other. Is `Alternative:` or `Suggestion:` better? But when does `Suggestion` overlap with the warning level suggestion?

**Comment by justinwoo** [comment]
Seems like there's quite a lot of overlapping places, so one approach I've tried is to remove "Why not" altogether and replace it with "You might want", as both `You might want:` and `You might want to remove it.` work as non-interrogatives

**Comment by justinwoo** [comment]
Pushed up the changes. Removal suggestion is longer now but should be fine.

File: src/CC.hs [hunk 1]
```diff
@@ -99,7 +99,7 @@ fromIdea Idea{..} = Issue
         , T.pack from
         , "```"
         , ""
-        , "Why not"
+        , "You might want"
```
Comment (ndmitchell) [inline src/CC.hs]: Did this one get forgotten?

**Comment by justinwoo** [comment]
I think "perhaps you should" is the common form in US and elsewhere, whereas the first is less familiar.

**Comment by ndmitchell** [comment]
Makes sense - thanks for the cleanup!

**Pull request merged**